lib/conversion.py

```diff
@@ -557,6 +557,9 @@ def c13(site_folder, samigo_soup):
 # C14 Calculated questions - Scientific notation
 #  in: site_folder
 def c14(site_folder, samigo_soup):
+        data = {
+            'calculated': set(),
```

Comment:
Please see https://jira.cilt.uct.ac.za/browse/AMA-752?focusedCommentId=222728&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-222728 about the structure of data that we return from conversion checks. Please update that first in a separate PR, then update this PR to match.